crates/sncast/src/response/print.rs

```diff
@@ -62,123 +68,150 @@ impl Display for OutputValue {
     }
 }
 
-pub fn print_command_result<T: CommandResponse>(
-    command: &str,
-    result: &mut Result<T>,
-    numbers_format: NumbersFormat,
-    output_format: &OutputFormat,
-) -> Result<()> {
-    let mut output: OutputData = vec![];
-    output.push((
-        String::from("command"),
-        OutputValue::String(command.to_string()),
-    ));
-    output.extend(result_as_output_data(result));
-    let formatted_output = output
-        .into_iter()
-        .map(|(k, v)| (k, apply_numbers_formatting(v, numbers_format)))
-        .collect();
-
-    for val in pretty_output(formatted_output, output_format)? {
-        match result {
-            Ok(_) => println!("{val}"),
-            Err(_) => eprintln!("{val}"),
+impl From<Value> for OutputValue {
```

Comment:
`NIT`: It's nice that you care about quality of code 😄, but I would do the refactor in a separate PR (Don't change this now, but it would be nice in the future). 

From the review perspective, it's hard to understand why some changes were made. Since it's not changing any behavior here, and it's not directly related to the PR topic, a separate refactor PR would be more appropriate.